include/openssl/x509.h

```diff
@@ -39,6 +39,13 @@
 extern "C" {
 #endif
 
+
+/* Flags for X509_get_signature_info() */
+/* Signature info is valid */
+# define X509_SIG_INFO_VALID     0x1
+/* Signature is suitable for TLS use */
+# define X509_SIG_INFO_TLS       0x2
```

Comment:
I might think about adding "_OK" to this flag's name; I'm not sure whether I would end up wanting to actually add it, though.